library/include/internal/rocblas-functions.h

```diff
@@ -25065,16 +25047,6 @@ ROCBLAS_EXPORT rocblas_status rocblas_set_workspace(rocblas_handle handle, void*
  ******************************************************************************/
 ROCBLAS_EXPORT bool rocblas_is_managing_device_memory(rocblas_handle handle);
```

Comment:
Can doxygen for rocblas_set_workspace indicate that this will imply the user is managing device memory and so rocblas_is_managing_device_memory will return false.  If that is correct of course